### ISSUE TITLE
Can't success in "Request Identify WMP"

### ISSUE THREAD

**Issue by stefanomazza** [Flafla2/Unity-Wiimote#53]
I'm trying to use my WiiMote (with MotionPlus inside), but I've found these problems:
- LEDs are costantly flashing, even if I select numeric buttons 
- The "Request Identify WMP" can't find my wiimote (if I try to press that multiple times, it says "the request is pending")

What can I do?

**Comment by dbqt** [comment]
Is your wiimote able to be connected to your computer at all? Without the use of the API.

**Comment by stefanomazza** [comment]
Yes, I've already connected the wiimote with bluetooth, and the API works, but only for the accelerometer line and for the buttons. I need the rotating feature of 3D wiimote model, but how can I use it if the API doesn't find the WMP?

**Comment by dbqt** [comment]
Maybe it doesn't support WMP integrated with the controller? I don't have one, I only have the separate WMP extension and it works just fine, so I can't test your specific case. 

Maybe @Flafla2 can help you out here.

**Comment by stefanomazza** [comment]
I don't think so... [Here](https://www.youtube.com/watch?v=co7xggFfE94) @Flafla2 used the same wiimote I have. What is the problem? @Flafla2

**Comment by danger9819** [comment]
I'm not quite sure if this is intentional design or not, but it seems that the Request and Respond Identify WMP sets wmp_attached to true specifically for just the Wii Motion Plus extension, and not Wii Remote Pluses. For the latter, you can check if the remote type is WIIMOTEPLUS, and then directly call ActivateWiiMotionPlus() without checking wmp_attached. This does work, but it warns when this happens which makes me wonder whether this is a bug.
